src/directives/mwlCalendarDay.js

```diff
@@ -8,6 +8,10 @@ angular
 
     var vm = this;
 
+    vm.timePosition = vm.timePosition ? vm.timePosition : 'default';
+    vm.timeHidden = vm.timePosition === 'hidden';
+    vm.timePositionOffset = vm.timePosition !== 'default' ? 0 : 60;
```

Comment:
In order for this to work it needs to be inside the `refreshView()` function, as with ng 1.6+ these inputs wont be initialised on the components construction.